### PR TITLE
Allow plists to be used as a localised resource

### PR DESCRIPTION
**Pull request by mikezucc**
----- Issue #39 

The following directory structure does not get built into the xcodeproject as a localised resource:

screen shot 2018-09-19 at 4 10 51 pm

There was a hard filter to prevent plists from being ingested as resources.

### CODE REVIEW

File: Sources/XCHammer/XcodeTarget.swift [hunk 1]
```diff
@@ -345,8 +345,7 @@ public class XcodeTarget: Hashable, Equatable {
 
     func isAllowableXcodeGenSource(path: Path) -> Bool {
         // These files are handled in other ways.
-        // FIXME: remove filtering plist ( these could be used in source file )
-        return path.extension != "plist" && path.extension != "mobileprovision"
```
Comment (jerrymarino) [inline Sources/XCHammer/XcodeTarget.swift]: I can't remember exactly why, but there was some reason `plist` is here. We should see the output of `make run_force`, and additionally update it per your commit.

**Comment by mikezucc** [comment]
screen shot 2018-09-20 at 3 54 58 pm

**Comment by mikezucc** [comment]
I love these mixed naming conventions Apple lmao

File: Sources/XCHammer/XcodeTarget.swift (hunk 1 above)
Comment (mikezucc) [reply Sources/XCHammer/XcodeTarget.swift]: Per our discussion, we verified that `make run` and `make test` succeed after fixing an issue with plist number formatting

**Comment by jerrymarino** [comment]
@mikezucc this LGTM - minor suggestion, rename `Jerry-extension` to something more descriptive of what it is.

**Pull request merged**